File: atcoder.jp/contests/abc218/tasks/abc218_b/main.ts
import { readFileSync } from "fs";
let inputLinesIndex = 0;
const inputLines = readFileSync("/dev/stdin", "utf8").split("\n");
const input = () => inputLines[inputLinesIndex++];
const print = (...messages: any) => console.log(...messages);

const asciiLowercase = "abcdefghijklmnopqrstuvwxyz";

// input functions -------------------------------------------------------------
const inputNumberArray = (): number[] => input().split(" ").map(Number);
// input functions -------------------------------------------------------------

const main = () => {
  const P = inputNumberArray();

  const ans = P.map((p) => asciiLowercase[p - 1]);
  print(ans.join(""));
};

main();

File: atcoder.jp/contests/abc218/tasks/abc218_b/main.py
from string import ascii_lowercase


# input functions --------------------------------------------------------------
def input_int_list() -> "list[int]":
    return list(map(int, input().split()))


def main() -> None:
    P = input_int_list()

    ans = [ascii_lowercase[p - 1] for p in P]
    print(*ans, sep="")


main()
